src/index.html

```diff
@@ -8,6 +8,43 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+
+      <img src="images/imac.jpeg"
+        alt="APPLE A1419 iMac 27' Retina 5K Monoblock (MNED2UA/A)">
```

Comment:
```suggestion
      <img 
        src="images/imac.jpeg"
        alt="APPLE A1419 iMac 27' Retina 5K Monoblock (MNED2UA/A)"
      >
```

Reply:
still not fixed
<img width="580" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/70220667/43b36996-00d4-402f-8eeb-07c8551051f9">